docs/coverage.md

```diff
@@ -0,0 +1,82 @@
+# Coverage
```

Comment:
Link this document from `docs/internals.md`.

Reply:
Fixed